src/views/tutorial-view/index.tsx

```diff
@@ -76,12 +76,13 @@ const LayoutContentWrapper = ({
 			 * TODO: What should we do if this errors?
 			 * https://app.asana.com/0/1202097197789424/1202599138117878/f
 			 */
-			getCollectionViewSidebarSections(product, collectionCtx.current).then(
-				(result: CollectionCategorySidebarSection[]) => {
-					hasLoadedData.current = true
-					setCollectionViewSidebarSections(result)
-				}
-			)
+			getCollectionViewSidebarSections(
+				product.slug,
```

Comment:
Have changed the arg from `product` to `product.slug` since that's all that was being used.